packages/bpk-component-card/src/BpkCardWrapper.tsx

```diff
@@ -31,27 +37,127 @@ type Props = {
   className?: string | null;
   backgroundColor: string;
   header: ReactNode;
+  body?: {
```

Comment:
body as a name I think might be confusing to consumers, maybe we could call this `expandContent` so its clear where this would be.

wdyt?